semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbSignatures.java

```diff
@@ -11,12 +11,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static com.sourcegraph.semanticdb_javac.Debugging.pprint;
-
 public final class SemanticdbSignatures {
   private final GlobalSymbolsCache cache;
   private final LocalSymbolsCache locals;
 
+  private static final Semanticdb.Type UnresolvedTypeRef =
```

Comment:
I believe the standard naming convention for static variables is s/UnresolvedTypeRef/UNRESOLVED_TYPE_REF/

(GitHub is not allowing me to write suggestions for some reason)